repos/system_upgrade/el7toel8/actors/ziplcheckmultiplerescueentries/tests/test_ziplcheckmultiplerescueentries.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from leapp import reporting
+from leapp.libraries.actor import ziplcheckmultiplerescueentries
+from leapp.libraries.actor.ziplcheckmultiplerescueentries import inhibit_if_multiple_rescue_entries_present
+from leapp.libraries.common.testutils import create_report_mocked, CurrentActorMocked
+from leapp.libraries.common.config import architecture
+from leapp.libraries.stdlib import api
+from leapp.models import BootEntry
+from leapp.snactor.fixture import current_actor_context
+
+
+def test_inhibition_multiple_rescue_entries_present(monkeypatch):
+    """Tests whether the upgrade process is inhibited when multiple rescue boot entries are present."""
+    mocked_report = create_report_mocked()
+    monkeypatch.setattr(reporting, 'create_report', mocked_report)
+
+    boot_entries = [
+        BootEntry(title='entry_1'),
+        BootEntry(title='entry_1_Rescue'),
+        BootEntry(title='entry_2'),
+        BootEntry(title='entry_2_rescue-ver2.3'),  # Typically the `rescue` substring surrounded
```

Comment:
```suggestion
        BootEntry(title='entry_2_rescue-ver2.3'),  # Typically is the `rescue` substring surrounded
```

Reply:
This is fixed in the latest version of this PR.